components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java

```diff
@@ -280,7 +280,7 @@ void setStartValue(double startValue)
 
 	/**
 	 * Sets the value of the end knob, value in the range
-	 * <code>]{@link #startValue}, {@link #maximum}]</code>.
+	 * <code>]{@link #startValue}, {@link #maximum}]  (one based) </code>.
```

Comment:
I guess it's an US-Europe thing, at least Germany typically also uses the square bracket notation.